sql/moz-fx-data-shared-prod/accounts_backend_derived/users_services_first_seen_v1/schema.yaml

```diff
@@ -0,0 +1,16 @@
+fields:
+- name: user_id_sha256
+  type: STRING
+  mode: NULLABLE
+- name: service
+  type: STRING
+  mode: NULLABLE
```

Comment:
Should we add `description` key here to provide information about this field. For example, that the value is actually corresponding to the service's id rather than the name. Could also include information that the name mapping is done via the view. Here's an example of adding a description to a field via the schema.yaml:
https://github.com/mozilla/bigquery-etl/blob/459c22e625aadf17c6de90af4fc5255dd41cf91e/sql/moz-fx-data-shared-prod/accounts_db_external/fxa_signin_codes_v1/schema.yaml#L5